scripts/redcap/purge_outdated.py

```diff
@@ -179,8 +202,15 @@ def main(api, args):
         sys.exit()
 
     marks = main(redcap_api, args)
+
+    # Log dataframe by row here, one for exceeds, and another for precedes
+    log_dataframe_by_row(marks[marks['exceeds']], uid_template=UID_TEMPLATE, message="Entry form exceeds visit date.",
+        description="Listed is an entry form that exceeds the visit date by 120 days.")
+    log_dataframe_by_row(marks[marks['precedes']], uid_template=UID_TEMPLATE, message="Entry forms precedes visit date.",
+        description="Listed is an entry form that precedes the visit date.")
+
     if args.output:
         marks.to_csv(args.output)
     else:
-        with pd.option_context('display.max_rows', -1):
```

Comment:
`-1` used to be equivalent to the result you now get when you supply `None` - i.e. display all rows. I'd suggest `None` is more useful.